api/schema.go

```diff
@@ -46,3 +42,8 @@ func (h *SchemaHandler) Get(ctx *gin.Context) {
 
 	ctx.JSON(http.StatusOK, schema)
 }
+
+type Schema struct {
```

Comment:
In order to pass the docs generation (swagger spec files), moved the Schema type to be a first-class citizen in api package.

Reply:
Confirmed with Jeff to be OK.